vlemish/3SumSolution.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace _3Sum
+{
+    public class _3SumSolution
+    {
+        // The easiest way to solve the problem is to decrease problem to 2Sum problem
+        // we can do it by creating a dictionary with elements for which we're going to search further.
+        public IList<IList<int>> FindTriples(int[] nums)
+        {
+            // if length is less than 3 -> no triples can be found
+            if (nums.Length < 3)
+            {
+                return new List<IList<int>>();
+            }
+
+            Array.Sort(nums);
```

Comment:
I sorted each triplet to compare but could have sorted the whole thing first like you did. :)